src/schema-generator/conditional-render.ts

```diff
@@ -55,9 +58,13 @@ const parseChildrenConditionalRenders = (
 				};
 			}
 		} else {
-			const idsToDelete = [id, ...listAllChildIds(componentSchema.children)];
-			idsToDelete.forEach((idToDelete) => {
-				delete parsedYupSchema[idToDelete];
+			const hiddenFieldIdList = [id, ...listAllChildIds(componentSchema.children)];
+			hiddenFieldIdList.forEach((hiddenFieldId) => {
+				parsedYupSchema[hiddenFieldId] = Yup.mixed().test(
+					"empty",
+					ERROR_MESSAGES.UNSPECIFIED_FIELD(hiddenFieldId),
+					(value) => ValueHelper.isEmpty(value)
```

Comment:
may have to use `isNil` instead. because with `{ field1: "do not show wrapper", field2: "" }` field2 passes the empty check since ValueHelper doesn't know it's supposed to be a number type

Reply:
added check for null values too